content/blog/2025-06-02-invited-speakers/index.en.md

```diff
@@ -0,0 +1,124 @@
+
+## Invite them clearly, in advance
+
+What are you inviting the person for exactly? 
+Your initial contact should state it explicitly, introducing the event in one or a few sentences and providing a link to the next or, if needed, past editions.
+
+Example questions your invitation should answer: 
+- How long will the talk be? 
+- Do you have a topic in mind or does the speaker have free reign? 
+- Will it be online? 
+- Will travel and accommodation be covered? Do state the budget.
+- Will you pay for it or will it be by reimbursement? Give details of the process. Ideally, be prepared to either do the bookings on behalf of the speaker to prevent their having to advance funds, or to let the speaker do the bookings, based on their preferences and constraints. See also the post section about logistics!
+- Will there be a speaker honorarium? If there is, it should be the same for all speakers.
+
+Don’t hesitate to throw in a compliment regarding why you are inviting this person, but do not overdo it.
+Explain why you invited the person: show that you know and respect their work.
+If someone recommended the speaker to you, only state it if you have the agreement of the recommender.
+
+Even if you made an effort to invite someone from a minority (and yes you should make an effort to have diversity in your speaker panel, we’re in the XXIst century)… do not tell this person.
+
+Send the invitation in advance, especially if the thing requires travelling, but even if not. 
+Be mindful of timelines, challenges, and costs with [procuring travel visas](https://yabellini.netlify.app/blog/2023-05-06-visa/). 
+Does the speaker have sufficient notice, and can organizers provide the required documentation?
+
+Do not list the person on a public draft schedule, or a Google Doc you share with all invited people, before you’ve heard from them. 
+Keep your hopes private!
+
+## Communicate regularly
+
+When the person answers, do not delay your own answer. 
+Be enthusiastic if they said yes, and understanding if not!
+
+You can communicate to indicate you don’t know about some things yet but that you’re working on them. 
+It is much better to answer about uncertainty, rather than not to answer.
+
+## Respect your guest
+
+If the guest has to adapt, or cancel, their participation in your event due to health, safety or other concerns, just accept this. 
+Do not push them to participate in a certain way disregarding their safety. 
+It is very unlikely the person is faking their concern or health issue.
+
+Ask about your guest’s needs, for instance but not limited to: room accessibility, childcare, lactation/breastfeeding space, food allergies/preferences, room for their family in the hotel room, etc. 
+Even better, have public information about how these things are handled at your event, as non guests will have similar problems!
+
+Ask the speaker for a title, abstract, bio and picture.
+Do not create or edit those without their explicit approval.
```

Comment:
```suggestion
Do not create or edit those without their explicit approval.
Once this information is publicly available on a website or otherwise, send it to the speaker.
```

Reply:
Perhaps here we can add about the social media and bio information. (I made a comment below)